package.json

```diff
@@ -52,7 +52,7 @@
     "shelljs": "^0.8.3",
     "ts-jest": "^24.0.2",
     "tslint": "^5.18.0",
-    "typescript": "^3.5.3",
+    "typescript": "^3.8.0-dev.20200201",
```

Comment:
https://github.com/microsoft/TypeScript/releases/tag/v3.8-rc

Reply:
I'll wait for 3.8.0 (should be yesterday)